target/linux/ath79/dts/ar7161_ruckus_gd11.dtsi

```diff
@@ -131,7 +131,7 @@
 &pcie0 {
 	status = "okay";
 
-	ath9k0: wifi@0,11 { /* 2.4 GHz */
+	ath9k0: wifi@11,0 { /* 2.4 GHz */
 		compatible = "pci168c,0029";
 		reg = <0x8800 0 0 0 0>;
```

Comment:
Any idea how this `11` come from?

Reply:
0x8800 shift right 11?

Reply:
if my math is right, `0x900 >> 11 == 0x12`.

Reply:
Oh, I confused decimal and hexadecimal in the calculator. Just forget it.